lit_gpt/config.py

```diff
@@ -188,6 +188,53 @@ def norm_class(self) -> Type:
     ),
 ]
 
+
+##########################
```

Comment:
Why did you choose to move this?

Reply:
So StableCode is placed right after StableLM.